.chloggen/msg_update-docs-for-component.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
I don't think this qualifies as a breaking change, since nothing is changing here.

Reply:
Sorry, I was thinking from this perspective the perspective of existing components that didn't expect to be `Stop` without being `Start`ed.

Reply:
I don't believe that a doc string change needs a changelog entry at all. I would like to keep the changelog limited to user facing changes. Doc updates shouldn't be included IMO. Otherwise it'll be hard to find significant changes in the list

Reply:
Let me remove this :)